task/upload.go

```diff
@@ -539,23 +549,26 @@ func uploadTaskOutputLocations(tctx *TaskContext) ([]OutputName, []clients.Outpu
 	return outputNames, outputLocations, nil
 }
 
-func outputLocations(outURL string, relativePath string, autoMp4s bool) ([]OutputName, []clients.OutputLocation, error) {
+func outputLocations(outURL string, hls string, hlsRelPath string, mp4 string, mp4RelPath string) ([]OutputName, []clients.OutputLocation, error) {
 	url, err := url.Parse(outURL)
 	if err != nil {
 		return nil, nil, fmt.Errorf("error parsing object store URL: %w", err)
 	}
-	// source segments cannot be stored in web3.storage
-	sourceSegments := !strings.HasPrefix(outURL, "w3s")
 	names, locations :=
 		[]OutputName{OutputNameOSPlaylistHLS},
 		[]clients.OutputLocation{
 			{
 				Type: "object_store",
-				URL:  url.JoinPath(hlsRootPlaylistFileName(relativePath)).String(),
+				URL:  url.JoinPath(hlsRelPath).String(),
+				Outputs: &clients.OutputsRequest{
+					HLS: hls,
+				},
+			},
+			{
```

Comment:
Just double-checking: Does catalyst support multiple `output_locations` provided on the request now? Previously it only read the first one and checked the outputs

Reply:
Yes, it will support it after this PR is merged: https://github.com/livepeer/catalyst-api/pull/546